### PR TITLE
Add listening on invoices topic and post messages

### PR DESCRIPTION
**Pull request by ashkan18**
# Change
Start listening on new `invoices` topic and post to slack when getting new event.

# Migrations 
Add new `invoices` to `topics`
Remove `bidding` now that we've moved to `auctions`

### CODE REVIEW

**Comment by ashkan18** [comment]
merging this so we can see this in Staging, @gnilekaw will address any comments in separate PR 🙌

File: lib/views/invoice_slack_view.ex [hunk 1]
```diff
@@ -0,0 +1,29 @@
+defmodule Aprb.Views.InvoiceSlackView do
+  import Aprb.ViewHelper
+
+  def render(event) do
+    partner_data = fetch_partner_data(event["properties"]["partner_id"])
+    %{
+      text: ":money_with_wings: Invoice #{event["object"]["display"]} #{event["verb"]}",
+      attachments: "[{
```
Comment (gnilekaw) [inline lib/views/invoice_slack_view.ex]: What about refactoring the JSON into an object that's then Poison-encoded? Something like:

```ex
data = [ %{ text: "...", attachements: "..." ] 
data |> Poison.encode
```

Comment (ashkan18) [inline lib/views/invoice_slack_view.ex]: I remember trying that originally and had issues with it! but im totally down for trying it again and clean up this string mess hehehe.

Comment (gnilekaw) [inline lib/views/invoice_slack_view.ex]: There's a lot of handwaving with my snippet. I think the second line would actually return something like `{ :ok, "[\"I'mmajson\"}]" }`. Happy to pair on it!

Comment (ashkan18) [inline lib/views/invoice_slack_view.ex]: lets do it!

**Pull request merged**